src/Microsoft.Health.Dicom.Core/Features/CustomTag/GetCustomTagsService.cs

```diff
@@ -30,14 +32,28 @@ public async Task<GetCustomTagResponse> GetCustomTagAsync(string tagPath, Cancel
         {
             string internalTagPath = _dicomTagParser.ParseFormattedTagPath(tagPath);
```

Comment:
Is this checking validation of tagPath including null/empty and throwing validation exception?